src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java

```diff
@@ -102,12 +106,16 @@ public void setAveragePower(final double averagePower) {
         this.averagePower = averagePower;
     }
 
+    private void computeAveragePower(final Duration deltaTime) {
```

Comment:
can you please apply my previous comment here?

Reply:
Yep I didn't read it, on it now.